templates/simpletimer.timer.epp

```diff
@@ -0,0 +1,14 @@
+<%- |
+  Hash[String[1],Variant[String[1],Integer],1] $timings
+| %>
+[Unit]
+Description=Puppet created timer
```

Comment:
Would it make sense to make it a parameter? Perhaps with this as a default value.